scripts/kind-e2e/e2e.sh

```diff
@@ -86,6 +86,15 @@ function kind_import_images() {
         kind --name cluster${i} load docker-image submariner:local
         kind --name cluster${i} load docker-image submariner-route-agent:local
     done
+
+    if [ ! -z "${global_CIDRs[cluster1]}" ]
+    then
+        docker tag quay.io/submariner/submariner-globalnet:dev submariner-globalnet:local
```

Comment:
please use $VERSION instead of dev, you may need to rebase this patch to latest master for that to work though.
```suggestion
        docker tag quay.io/submariner/submariner-globalnet:$VERSION submariner-globalnet:local
```

Reply:
Done